<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemeWindow</class>
 <widget class="QWidget" name="SchemeWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SchemeWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>窗口位置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>402</width>
            <height>207</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashLine</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>Local</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Selected Item</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Overview</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Drones</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Panel</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Undock</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>窗口</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>Local</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>Selected Item</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>Overview</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>Drones</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>Panel</string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>Undock</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbwdetected">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>检测</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Wrap to Within：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbWrap">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Within 0 m</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 10 km</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 20 km</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 30 km</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 50 km</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 70 km</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 100 km</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbShield">
          <property name="text">
           <string>盾抗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbArmour">
          <property name="text">
           <string>甲抗</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>366</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbApply">
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
